api-spec/README.md

```diff
@@ -54,9 +54,9 @@ The [`definitions` folder](./definitions/) contains the YAML files that will imp
 
 See [yaml-files](https://www.npmjs.com/package/yaml-files) for the syntax to import YAML fragments.
 
-### Fragments
+### Extensions

```

Comment:
Was the change from fragments to extensions intended to be in this PR as well?

Reply:
I guess that was changed as the folder is called extensions, not fragments.